src/Data/Avro/Deconflict.hs

```diff
@@ -57,8 +57,8 @@ resolveSchema e d v
 -- The writer's symbol must be present in the reader's enum
 resolveEnum :: Type -> Type -> T.Value Type -> Either String (T.Value Type)
 resolveEnum e d val@(T.Enum _ _ _txt) = Right val
-   -- | txt `elem` symbols d = Right val
-   -- | otherwise = Left "Decoded enum does not appear in reader's symbol list."
+  -- --  | txt `elem` symbols d = Right val
```

Comment:
because `-- |` happens to be a `haddock` instruction and `cabal` would complain when preparing a package